.github/workflows/test-ci.yml

```diff
@@ -139,7 +139,7 @@ jobs:
         run: |
           cd $RUNNER_WORKSPACE
           rm -rf finufft
-          git clone https://github.com/chaithyagr/finufft --branch chaithyagr/issue306
+          git clone https://github.com/flatironinstitute/finufft
```

Comment:
Is there no `pip install`?

Reply:
As the development of cufinufft is very active, I prefer to track directly the repo.

Reply:
We might also need to test against the latest release version as that's what users use.